src/main/java/Event.java

```diff
@@ -0,0 +1,16 @@
+public class Event extends Task {
+    protected String start;
```

Comment:
Perhaps change the start and end variable names to something more specific, like startTime.